### PR TITLE
Topic 3: Translate backend templates files

### PR DESCRIPTION
**Pull request by imenattatra**
## What type of PR is this? 

Related to the translation project : https://github.com/metr-systems/backlog/issues/3476

- [x] Feature - follow up

## Description

Things you need to know : 
- we temporary revert [this commit ](https://github.com/metr-systems/redash/commit/2f4362a68dbf7b4c4917da3152bdb6dc3f4e56fc) so that we stay conform to the original redash but we will get it back later before we deploy to our redash, it is just that we needs the original translations so that we can open a PR to original redash later
- we translate all files that are under "templates" folder in redash backend 

## How is this tested?

- [x] Unit tests (pytest)
- [ ] Manually - [WIP ... updates may happen if anything noticed, but i am openning a PR from now]

## Next steps

-> Add flag feature for choosing the language (en/de)
-> Do more manual tests on staging and deploy (make sure we deploy after we get back [that commit ](https://github.com/metr-systems/redash/commit/2f4362a68dbf7b4c4917da3152bdb6dc3f4e56fc))
-> Finish the rest of the translation

Redash spricht bald Deutsch !

These changes are currently deployed to staging

### CODE REVIEW

**Review by helenalebreton — APPROVED** [review]

File: redash/templates/emails/invite.html [hunk 1]
```diff
@@ -2,36 +2,18 @@
...
-            <a class="intercom-h2b-button" target="_blank" href="{{ invite_url }}" style="background: #0071b2; border: none; border-radius: 3px; color: white; display: inline-block; font-size: 14px; font-weight: bold; outline: none !important; text-decoration: none">
-                Konto vervollständigen
-            </a>
-        </td>
-    </tr>
-</table>
-<p class="intercom-align-left" style="line-height: 1.5; margin: 0 0 17px; text-align: left !important" align="left">
-    Der Button funktioniert nicht? Dann kopieren Sie bitte diesen Link in Ihren Browser: {{ invite_url }}
-</p>
+<p class="intercom-align-left" style="line-height: 1.5; margin: 0 0 17px; text-align: left !important" align="left">Hi {{ invited.name }},</p>
```
Comment (thiagogds) [inline redash/templates/emails/invite.html]: Should we translate also the "Hi"?

**Review by thiagogds — APPROVED** [review]
Wow, that's a lot of translations. Nice work

File: redash/templates/emails/invite.html (hunk 1 above)
Comment (imenattatra) [reply redash/templates/emails/invite.html]: nice catch i did not see it hehe

Comment (imenattatra) [reply redash/templates/emails/invite.html]: I fixed it in the last commit :D , thanks

**Pull request merged**